week-1/exercises/initials.js

```diff
@@ -4,3 +4,5 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first 3 characters of each string in upper case
 // Log the variable in each case
+let initials = firstName.slice(0 , 3). toUpperCase()+ middleName.slice(0 , 3). toUpperCase()+ lastName.slice(0, 3). toUpperCase();
```

Comment:
An alternative way to do this is `let initials = (firstName.slice(0, 3) + middleName.slice(0, 3), lastName.slice(0, 3)).toUpperCase()`